src/components/Card/Card.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+import { Image, StyleSheet, View } from "react-native";
+
+import { styled } from "nativewind";
+
+import { CardType } from "./Card.types";
+
+const Card: React.FC<CardType> = ({ type, cardColor, imageURL, children }) => {
+	const generateCardBaseStyle = (
+		cardColor: { backgroundColor?: string | undefined } | undefined,
+		type: string | undefined
+	): object => {
+		if (type === "elevated") {
+			return {
+				...style.baseCard,
+				...style.elevatedCard,
+				...cardColor
+			};
+		}
+
+		if (type === "outlined") {
+			return {
+				...style.baseCard,
+				...style.outlinedCard,
+				borderColor: cardColor?.backgroundColor
+			};
+		}
+
+		return {
+			...style.baseCard,
+			...style.containedCard,
+			...cardColor
+		};
+	};
+
+	const generatedStyle = generateCardBaseStyle(cardColor?.[0], type);
```

Comment:
look at how I did the style for the text, let's use composition instead of using spread ooperator to merge styles